docs/index.rst

```diff
@@ -45,6 +45,7 @@ Getting Started
     6. :doc:`Register a workflow on Dockstore <getting-started/dockstore-workflows>`
     7. :doc:`Hosted tools and workflows <getting-started/hosted-tools-and-workflows>`
 * :doc:`Services (beta) <getting-started/getting-started-with-services>`
```

Comment:
Yikes, I see we already had services described as beta. Maybe my memory is wrong, but I thought we had a story time a long time where we decided to label it Preview on the web site, with part of the reason being that we're not allowed to release beta software under NIST 800-53. @david4096 Does that sound right?

If so, maybe we just failed to update the docs.

Reply:
Yeah, I copied the beta wording from the services documentation. I did find this ticket https://ucsc-cgl.atlassian.net/browse/SEAB-697, but I couldn't find the story time where we made the decision. Either way, it seems like it should be updated to preview because that's what we use in the UI

Reply:
> Updated instances of `beta` to `preview`, including for other features. I didn't touch the usage of beta in the release notes though, such as [here](https://github.com/dockstore/dockstore-documentation/blob/2e84d4940a5522cffb0e6aedaa6a0ca0fa383bd1/docs/news/2018-09-12-Dockstore-1.5.0.rst?plain=1#L27)

That looks fine; the `beta` usages left are from pre-compliance days.